server/sonar-web/src/main/js/app/components/nav/settings/SettingsNav.tsx

```diff
@@ -77,7 +80,7 @@ export default class SettingsNav extends React.PureComponent<Props> {
   };
 
   render() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The function render has a cyclomatic complexity of 14 which is higher than the threshold of 10 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)